src/site/content/en/blog/performance-measure-memory/index.md

```diff
@@ -0,0 +1,356 @@
+---
+title: Monitor your web page's total memory usage with `performance.measureMemory()`
+subhead: >
+  Learn how to measure memory usage of your web page in production to detect regressions.
+description: >
+  Learn how to measure memory usage of your web page in production to detect regressions.
+date: 2020-04-07
+authors:
+  - ulan
+hero: hero.jpg
```

Comment:
@ulan please optimize this with [squoosh](squoosh.app) so that we keep the size of the repository down

Reply:
@kaycebasques thanks for the tip, reduces the size to 116K